.changeset/removal-of-database-cache-adapter.md

```
@@ -0,0 +1,17 @@
+---
+"@daiso-tech/core": minor
+---
+
+Removed `IDatabaseCacheAdapter`, `IDatabaseCacheTransaction`, and `ICacheData` contracts, along with their `databaseCacheAdapterTestSuite`.
+
+The `IDatabaseCacheAdapter` contract (`database-cache-adapter.contract.ts`) has been removed in favor of the simpler `ICacheAdapter` contract. This simplifies the cache adapter interface by eliminating the transaction-based database abstraction layer.
+
+### Changes:
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)